### PR TITLE
Reload prompt for plaintextPatches

### LINKED ISSUE #377: [Plugin installer] Prompt users to reload if plugin has plaintext patches
Since plaintext patches won't be applied until you reload Discord, the plugin installer should check if a plugin has plaintext patches, and if so, prompt users to reload (`window.location.reload` should do it) to properly activate the plugin.

### PR DESCRIPTION
**Pull request by bop34**
Closes #377

### CODE REVIEW

File: src/renderer/coremods/installer/util.ts [hunk 1]
```diff
@@ -246,6 +246,17 @@ export async function installFlow(
 
   await install(info);
 
+  if ("plaintextPatches" in info.manifest) {
+    const res = await modal.confirm({
```
Comment (asportnoy) [inline src/renderer/coremods/installer/util.ts:250]: Can you make this not block the function return?

Comment (bop34) [reply src/renderer/coremods/installer/util.ts:253]: I guess I could try something with events?

File: src/renderer/coremods/installer/util.ts [hunk 2]
```diff
@@ -246,6 +246,20 @@ export async function installFlow(
...
   await install(info);
 
+  if (
+    ("plaintextPatches" in info.manifest || "reloadRequired" in info.manifest) &&
+    info.manifest.reloadRequired
+  ) {
+    const res = await modal.confirm({
+      title: Messages.REPLUGGED_UPDATES_AWAITING_RELOAD_TITLE,
+      body: Messages.PLUGIN_INSTALL_PROMPT_BODY.format({ name: info.manifest.name }),
+      confirmText: Messages.REPLUGGED_INSTALLER_INSTALL_PROMPT_BODY,
```
Comment (asportnoy) [inline src/renderer/coremods/installer/util.ts:256]:
```suggestion
      confirmText: Messages.REPLUGGED_RELOAD,
```

File: src/renderer/coremods/installer/util.ts [hunk 3]
```diff
@@ -246,6 +246,23 @@ export async function installFlow(
 
   await install(info);
 
+  if (
+    ("plaintextPatches" in info.manifest || "reloadRequired" in info.manifest) &&
+    info.manifest.reloadRequired
+  ) {
```
Comment (asportnoy) [inline src/renderer/coremods/installer/util.ts:252]:
```suggestion
  if (
    "reloadRequired" in info.manifest
      ? info.manifest.reloadRequired
      : "plaintextPatches" in info.manifest
  ) {
```

**Review by asportnoy — CHANGES_REQUESTED** [review]

**Review by asportnoy — APPROVED** [review]

**Pull request merged**